### PR TITLE
Configuration

### PR DESCRIPTION
**Pull request by linnnsss**
Added new essay explaining config and reorganized DevOps Section.

### CODE REVIEW

**Review by linnnsss — COMMENTED** [review]
typos and capitalization fixed

File: docs/devops/configuration.md [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+# Configuration
+## Basics
+### accounts
+Account is a list that includes the addresses and amount of tokens initially allocated. The value of each node has must be consistent.
+| Value Type| Default Value| Unit | Optional |
+| --------- | ------------ | ---- | -------- |
+| Vec\<InitialAccount> | - | - |  False  |
```
Comment (ahonn) [inline docs/devops/configuration.md:15]: please try `Vec\<InitialAccount\>` to void syntax error

**Review by linnnsss — COMMENTED** [review]
<> format fixed

File: docs/devops/configuration.md [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+| --------- | ------------ | ---- | -------- |
+| Uint | 25000000 | - |  True  |
+  
+
+## Network
+### bootstraps
+  An array containing the list of bootstrap nodes.
+ | Value Type| Default Value| Unit | Optional |
+| --------- | ------------ | ---- | -------- |
+| Vec<> | - | - |  False  |
```
Comment (KaoImin) [inline docs/devops/configuration.md:91]:
```suggestion
| Vec\<String\> | - | - |  False  |
```

File: docs/devops/configuration.md [hunk 3]
```diff
@@ -0,0 +1,261 @@
...
+  Whether to enable metrics.
+   | Value Type| Default Value| Unit | Optional |
+| --------- | ------------ | ---- | -------- |
+| Bool | True | - |  True  |
+  
+### modules_level
+  The log output level defined by module.
+    | Value Type| Default Value| Unit | Optional |
+| --------- | ------------ | ---- | -------- |
+| HashMap<String, String> | - | - |  True  |
```
Comment (KaoImin) [inline docs/devops/configuration.md:212]:
```suggestion
| HashMap\<String, String\> | - | - |  True  |
```

File: docs/devops/configuration.md [hunk 4]
```diff
@@ -0,0 +1,261 @@
...
+| Bool | True | - |  True  |
+  
+### modules_level
+  The log output level defined by module.
+    | Value Type| Default Value| Unit | Optional |
+| --------- | ------------ | ---- | -------- |
+| HashMap\<String, String\> | - | - |  True  |
+
+```toml
+  modules_level = { "overlord::state::process" = "debug", core_consensus = "error" }
```
Comment (KaoImin) [inline docs/devops/configuration.md:215]:
```suggestion
modules_level = { "overlord::state::process" = "debug", core_consensus = "error" }
```

File: docs/devops/configuration.md [hunk 5]
```diff
@@ -0,0 +1,260 @@
...
+sidebar_position: 1
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+# Configuration
+## Basics
+### accounts
+Account is a list that includes the addresses and amount of tokens initially allocated. The value of each node has must be consistent.
+| Value Type| Default Value| Unit | Optional |
```
Comment (ahonn) [inline docs/devops/configuration.md:13]: The reason why the markdown table is not rendering is that there is no blank line before the table, causing it to be recognized as part of the previous paragraph. Adding a blank line after "12L" will display it correctly. Other tables in the current file have the same issue.

**Review by KaoImin — APPROVED** [review]

**Pull request merged**